contributors/guide/pull-requests.md

```diff
@@ -590,7 +592,31 @@ at once to that file.
 * Can the file be improved further?
 * Does the trivial edit greatly improve the quality of the content?
 
-## Fixing linter issues
+## Large or Automatic Edits
+
+Some tools make it very easy to create large Pull Requests, for example:
+- global search/replace
+- linters which automatically correct issues (see also next section)
```

Comment:
(parts of this are also redundant with the earlier section "Don't Open Pull Requests That Span the Whole Repository")

Reply:
True. I think they are a complimentary, so I cross-referenced them.

Reply:
https://github.com/kubernetes/community/compare/316bb29bf6c134ed9b0955971f6107ddf4f609b8..f67a105f9a1f379f46d0f7a554dffc3085baa44d